dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/api/ContextAPIServlet.java

```diff
@@ -107,12 +108,13 @@ public void testContextStorageProvider() {
         ContextStorageProvider provider = new MyContextStorageProvider();
         ContextStorage storage = provider.get();
         Context context = Context.current();
-        Scope scope = storage.attach(context);
-        assertNotNull(scope);
-        assertEquals(Scope.noop(), scope);
-        Context stored = storage.current();
-        assertEquals(context, stored);
-        scope.close();
+        try(Scope scope = storage.attach(context)){
+            assertNotNull(scope);
+            assertEquals(Scope.noop(), scope);
+            Context stored = storage.current();
+            assertEquals(context, stored);
+            scope.close();
+        }
```

Comment:
`scope.close()` not needed when using try-with-resources

The formatting around the `try` line doesn't look like it has spaces in the right places.